### PR TITLE
Add socials of Lawrence Roy and add as contributor

### PR DESCRIPTION
**Pull request by Lorns15**
#243 #244

### CODE REVIEW

**Comment by Lorns15** [comment]
added #17

File: .all-contributorsrc [hunk 1]
```diff
@@ -399,6 +399,15 @@
       "contributions": [
         "content"
       ]
+    },
+    {
+      "login": "Lorns15",
+      "name": "Lawrence Roy Quiling",
+      "avatar_url": "https://avatars.githubusercontent.com/u/39731070?v=4",
+      "profile": "https://github.com/Lorns15",
+      "contributions": [
```
Comment (Jansalvador1445) [inline .all-contributorsrc:408]: For what purpose?

Comment (Jansalvador1445) [reply .all-contributorsrc:408]: I think if not needed, please remove.

Comment (Jansalvador1445) [reply .all-contributorsrc:408]: Please revert this one na lang.

Comment (Lorns15) [reply .all-contributorsrc]: Na remove kona sir

**Comment by Jansalvador1445** [comment]
This is good! Merging it now! Congrats Lawrence! :D

**Comment by joshuadeguzman** [comment]
@allcontributors please add @Lorns15 for code, design

**Pull request merged**